Jenkinsfile

```diff
@@ -44,26 +31,19 @@ elifePipeline {
                 unstable_image.push()
             }
 
-            stage 'Push release image', {
-                if (isNew) {
-                    def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
-                    image.tag('latest').push()
-                    image.tag(candidateVersion).push()
-                }
-            }
+        }
+
+        elifeTagOnly { tagName ->
```

Comment:
this is executed in response to `git tag v0.0.2 && git push origin v0.0.2` but not in other cases